frog/params.rkt

```diff
@@ -40,3 +40,8 @@
 (define current-python-executable (make-parameter "python"))
 (define current-pygments-linenos? (make-parameter #t))
 (define current-pygments-cssclass (make-parameter "source"))
+(define current-responsive-images? (make-parameter #f))
+(define current-image-output-dir (make-parameter "img/resized"))
```

Comment:
I think this needs to change from `"img/resized"` to `"resized"`.  Elsewhere we'll append this to the value of the `(www/image-path)` function defined in `paths.rkt`.  In other words it's just the subdir of whatever the general images dir might be.